src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -96,21 +96,17 @@ scaleFeeLoad(FeeUnit64 fee, LoadFeeTrack const& feeTrack,
     // compatible. This function is an exception.
     auto lowestTerms = [](auto& a, auto& b)
     {
-        auto value = [](auto val)
+        auto const value = [](auto val)
         {
             if constexpr(std::is_arithmetic_v<decltype(val)>)
                 return val;
             else
                 return val.value();
         };
 
-        auto divide = [](auto& val, auto gcd)
+        auto const divide = [&value](auto& val, auto gcd)
```

Comment:
I'd remove this lambda. At this point the lambda is so simple that it's better to just replace the call to the lambda with this code directly.

Reply:
Done.